server/controllers/inventory/reports/changes.js

```diff
@@ -44,6 +44,8 @@ async function inventoryChanges(req, res, next) {
   const params = _.clone(req.query);
   const metadata = _.clone(req.session);
 
+  const { currencyId } = params;
```

Comment:
This is no longer defined.  It needs to be set to `req.session.enterprise.currency_id` to make it work.  Otherwise, looks good to me.

Reply:
Fixed